proofs/README.md

```diff
@@ -0,0 +1,51 @@
+# `turnkey_proofs`
+
+This crate contains utilities to parse and verify Turnkey enclave proofs. As outlined in [the Turnkey whitepaper](https://whitepaper.turnkey.com) there are two types of proofs:
+* App proofs, signing structured data with enclave ephemeral keys.
+* Boot proofs, which are proofs that a given enclave was provisioned correctly. Boot proofs reference, among other things, enclave ephemeral key. 
+
+## Boot proofs
+
+> 🚧 **Experimental**: Turnkey Boot proofs are not fully baked yet and may change significantly in the near future
+
+If you have a Turnkey organization you can request a Boot proof from any enclave. This boot proof is an attestation document from Amazon, signed by a root certificate associated with AWS Nitro Attestation PKI (located in [`aws_root.pem`](./static/aws_root.pem)). This top-level certificate can be downloaded from https://aws-nitro-enclaves.amazonaws.com/AWS_NitroEnclaves_Root-G1.zip.
+
+Resources on AWS Nitro Enclaves, attestations, and verifying attestations can be found at the following:
+
+- https://docs.aws.amazon.com/enclaves/latest/user/nitro-enclave.html
+- https://docs.aws.amazon.com/enclaves/latest/user/set-up-attestation.html
+- https://aws.amazon.com/blogs/compute/validating-attestation-documents-produced-by-aws-nitro-enclaves/
+- https://docs.aws.amazon.com/enclaves/latest/user/verify-root.html
+
+### Usage
+
+You may request a fresh attestation with the `turnkey` CLI (available [here](https://github.com/tkhq/tkcli)):
+```
+$ turnkey request --host api.turnkey.com --path /public/v1/query/get_attestation --body '{ "organizationId": "<your organization ID>", "enclaveType": "signer" }' --organization <your organization ID>
+
+{
+   "attestationDocument": "<base64-encoded attestation document -- copy and paste this into turnkey_attestation.txt>"
```

Comment:
to update?

Reply:
yes yes! Good catch.

Reply:
Addressed in https://github.com/tkhq/rust-sdk/pull/18/commits/5ecfbef882ef89ec5c77b2c2c18b7f820dfaf59c